tqdm.py

```diff
@@ -0,0 +1,85 @@
+import time
+
+PROGRESS_BAR_LEN = 20
+
+
+class Tqdm:
+    """
+    Iterable that prints status of other iterable objects
+    print examples:
+        10/20 : 5.03 : 1.99 : ##########.......... : 50%
+        9it : 6.78 : 1.33it/s
+    Attributes:
+        iterable: Iterable object to print his status
+        length: The length extracted from iterable
+    Methods:
+        __init__
+        __iter__
+    """
+
+    def __init__(self, iterable):
+        """
+        :param iterable: Iterable object to print his status
+        """
+        self.iterable = iterable
+        self.length = len(self.iterable) if hasattr(self.iterable, '__len__') else None
+
+    def __iter__(self):
+        """
+        :return: New iterator object
+        """
+        return TqdmIterator(iter(self.iterable), self.length)
+
+
+class TqdmIterator:
+    """
+    Iterator that prints the status of other iterators
```

Comment:
No need for copy-paste here.
No one will see this docstring because you rarely look at an iterator's documentation.